docs/user/algo_bc.md

```diff
@@ -0,0 +1,16 @@
+# Behavioral Cloning
+
+[[Paper](https://arxiv.org/abs/1605.08478)] [[Implementation](https://github.com/rlworkgroup/garage/blob/master/src/garage/torch/algos/bc.py)]
```

Comment:
please make this a full-on citation, including the arXiv URL. If this interrupts your page too might, you might find it useful to use a footnote RST environment (see https://recommonmark.readthedocs.io/en/latest/auto_structify.html#embed-restructuredtext and https://docutils.sourceforge.io/docs/user/rst/quickref.html#citations).

I'm unsure which one will look better.

Reply:
Honestly, I'm not even 100% sure this is *really* the right citation. It only really describes the log probability loss as one of its baseline methods, but I haven't been able to find a better citation.